ci_feature/setup.sh

```diff
@@ -31,6 +31,11 @@ mv $TMPDIR/omsbundle* $TMPDIR/omsbundle
 
 /usr/bin/dpkg -i $TMPDIR/omsbundle/100/omsagent*.deb
 /usr/bin/dpkg -i $TMPDIR/omsbundle/100/omsconfig*.deb
+
+#Assign permissions to omsagent user to access docker.sock
+sudo apt-get install acl
+sudo setfacl -m user:omsagent:rw /var/run/docker.sock
```

Comment:
Can you also please update this in ci_feature_prod/setup.sh ?

Reply:
Sure.